Softeq.XToolkit.HttpClient/Infrastructure/DefaultHttpClientProvider.cs

```diff
@@ -0,0 +1,24 @@
+using System.Net.Http.Headers;
+
+namespace Softeq.XToolkit.HttpClient.Infrastructure
+{
+    public class DefaultHttpClientProvider : IHttpClientProvider
+    {
+        public System.Net.Http.HttpClient CreateHttpClient()
+        {
+            var httpClient = DoCreateHttpClientInstance();
+            DoConfigureHttpClientInstance(httpClient);
+            return httpClient;
+        }
+
+        protected virtual System.Net.Http.HttpClient DoCreateHttpClientInstance()
+        {
+            return new System.Net.Http.HttpClient();
```

Comment:
namespace can be simplified with `using`

Reply:
Unfortunately, there's namespace intersection: System.Net.Http.HttpClient and Softeq.XToolkit.HttpClient. So, only using alias might help.